android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -399,11 +405,20 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
             }
         }
 
+        val offlineBehaviorParam = offlineBehavior.let {
+            when (it) {
+                "prefer_online" -> OfflineBehavior.PREFER_ONLINE
+                "require_online" -> OfflineBehavior.REQUIRE_ONLINE
+                "force_offline" -> OfflineBehavior.FORCE_OFFLINE
+                else -> OfflineBehavior.PREFER_ONLINE
```

Comment:
I realize we are following the same pattern we do elsewhere, but I think we should consider more strictly validating the input string parameters, rather than assuming a default value whenever the string doesn't match an expected value. Specifically, I think we should raise an exception here rather than defaulting to `PREFER_ONLINE`.

No change requested for this PR, but please consider making this change broadly throughout the RN code base in the future.